components/modal/check-props.js

```diff
@@ -18,6 +18,7 @@ if (process.env.NODE_ENV !== 'production') {
 			"assistiveText['closeButton']",
 			createDocUrl('assistiveText')
 		);
+		deprecatedProperty(COMPONENT, props.title, 'title', 'heading');
```

Comment:
It's not documented, but this should have the `createDocUrl('heading')` as the fifth parameter.